sound/soc/sof/intel/hda-stream.c

```diff
@@ -486,7 +486,7 @@ irqreturn_t hda_dsp_stream_threaded_handler(int irq, void *context)
 				continue;
 
 			/* Inform ALSA only in case not do that with IPC */
-			if (sof_hda->no_ipc_position)
+			if (sof_hda->ipc_nowakeup)
```

Comment:
see here as well, we have a complicated mix of hda->ipc_nowakeup and stream->no_period_wakeup. This makes no sense, we should really get rid of all the hda->ipc_nowakeup.